airbyte-ci/connectors/connectors_insights/src/connectors_insights/hacks.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from __future__ import annotations
```

Comment:
You don't need this as of Python 3.9, no?

Reply:
It was meant to not be required from 3.11, but it's is still. 

https://docs.python.org/3/whatsnew/3.11.html#pep-563-may-not-be-the-future